src/commands/create/groups/quickstart.ts

```diff
@@ -99,17 +162,17 @@ export default async (folderLocation: string, folderRelativePath: string, templa
     ...env,
     WALLET_PRIVATE_KEY: privateKey,
   };
-  const packageManager = await askForPackageManager();
+
+  const packageManager: PackageManagerType = isFramework(template, "Foundry") ? "forge" : await askForPackageManager();
   await setupTemplate(template, folderLocation, env, packageManager);
 
   successfulMessage.start(folderRelativePath);
-  Logger.info(`${chalk.magentaBright("Directory Overview:")}
-  - Contracts: /contracts
-  - Deployment Scripts: /deploy
-   
-${chalk.magentaBright("Commands:")}
-  - Compile your contracts: ${chalk.blueBright(packageManagers[packageManager].run("compile"))}
-  - Deploy your contract: ${chalk.blueBright(packageManagers[packageManager].run("deploy"))} 
-    - Tip: You can use the ${chalk.blueBright("--network")} option to specify the network to deploy to.`);
+
+  if (isFramework(template, "Foundry")) {
```

Comment:
Should we keep it as `template.framework === "Foundry"` in all places instead if `isFramework`? Tbh it's not really clear what that util does under the hood and since it's really simple maybe we can live without it?